packages/bpk-component-rating/src/BpkRating.js

```diff
@@ -73,15 +91,17 @@ const BpkRating = (props: Props) => {
       >
         <strong>{adjustedValue}</strong>
       </BpkText>
-      <div className={getClassName('bpk-rating__text')}>
+      <div className={getClassName('bpk-rating__text--container')}>
```

Comment:
Nitpick but in the past we've tried to avoid using 'container' for things like this and instead say 'wrapper'. I can't remember the exact reason but I think 'container' has some sort of meaning in Reactland. I'll see if I can find any more info

Reply:
https://medium.com/@learnreact/container-components-c0e67432e005

Also should it be `__container` because it's an element, not a modifier?

Reply:
Actually @shaundon I think it should be `bpk-rating__text-container` because it's not normal to have multiple layers of elements

Reply:
Actually @shaundon I think it should be `bpk-rating__text-container` because it's not normal to have multiple layers of elements

Reply:
Ah yeah you're right George